lib/eventFactory.js

```diff
@@ -12,6 +12,7 @@ const {
 } = require('screwdriver-data-schema').config.regex;
 const workflowParser = require('screwdriver-workflow-parser');
 const winston = require('winston');
+const _ = require('lodash');
```

Comment:
`lodash` is heavy. Since what we do is not too complicated, I would suggest to not include it.

Reply:
I think its okay, since we already have loadsh in the `screwdriver-cd/model`. Plus, its in the backend, I would have some reservation in the front end though.

```javascript
lib/eventFactory.js
15:const _ = require('lodash');

lib/pipeline.js
12:const _ = require('lodash');
```

Reply:
Include the module u needs